packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -133,8 +129,6 @@
 }
 
 .iui-input-decorator-icon {
-  margin: 0 var(--iui-size-s);
-
   &.iui-actionable {
```

Comment:
this css still needs to be removed

Reply:
Yup

Reply:
iui-actionable code will be moved to new pseudo element (todo)